src/Components/Components/src/Forms/EditContext.cs

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    /* Async validation plan
```

Comment:
ewww block comment. That's like the `async void` of comments.

Reply:
I'm curious about the reasoning, but in any case this is removed now.